fiftyone/operators/_types/pipeline.py

```
@@ -0,0 +1,119 @@
+    num_distributed_tasks: Optional[int] = None
+    params: Optional[Mapping[str, Any]] = None
+
+    def __post_init__(self):
+        if not self.operator_uri:
+            raise ValueError("operator_uri must be a non-empty string")
+
+        if (
+            self.num_distributed_tasks is not None
+            and int(self.num_distributed_tasks) < 1
+        ):
+            raise ValueError("num_distributed_tasks must be >= 1")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)